test/test_jit.py

```diff
@@ -3693,6 +3693,21 @@ def lstm(x, hx, cx, w_ih, w_hh, b_ih, b_hh):
                 self.assertTrue(type(block.paramNode()) == torch._C.Node)
         self.assertTrue(tested_blocks)
 
+    def test_export_opnames(self):
+        with torch.jit._disable_emit_hooks():
```

Comment:
Why is this used?

Reply:
It's a copy-paste. Removed for the updated PR.